### ISSUE TITLE
Sparrow Server - Exception in Application init method

### ISSUE THREAD

**Issue by caramel-chirping-decree** [sparrowwallet/sparrow#725]
Sparrow Server v 1.7.0 amd64 deb
22.04.1 LTS (Jammy Jellyfish) Server Edition
Local electrum server Fulcrum
Running whirlpool at the time

Sparrow.log relevant lines below.

```
2022-10-28 19:51:47,692 WARN [Thread-62] c.s.s.n.FeeRatesSource [null:-1] Error retrieving recommended fee rates from https://mempool.space/api/v1/fees/recommen
ded (Remote host terminated the handshake)
2022-10-28 19:56:55,490 WARN [Thread-94] c.s.s.n.ExchangeSource [null:-1] Error retrieving currency rates (Remote host terminated the handshake)
2022-10-28 19:56:55,513 ERROR [main] c.s.s.t.SparrowTerminal [null:-1] Exception in thread "main"
java.lang.RuntimeException: Exception in Application init method
        at javafx.graphics@18/com.sun.javafx.application.LauncherImpl.launchApplication1(Unknown Source)
        at javafx.graphics@18/com.sun.javafx.application.LauncherImpl.lambda$launchApplication$2(Unknown Source)
        at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NullPointerException: Cannot invoke "java.lang.Double.doubleValue()" because "<parameter2>" is null
        at com.sparrowwallet.sparrow@1.7.0/com.sparrowwallet.sparrow.terminal.wallet.WalletDialog.getFiatValue(Unknown Source)
        at com.sparrowwallet.sparrow@1.7.0/com.sparrowwallet.sparrow.terminal.wallet.UtxosDialog.lambda$exchangeRatesUpdated$17(Unknown Source)
        at com.googlecode.lanterna@3.1.1/com.googlecode.lanterna.gui2.AbstractTextGUIThread.processEventsAndUpdate(Unknown Source)
        at com.googlecode.lanterna@3.1.1/com.googlecode.lanterna.gui2.MultiWindowTextGUI.waitForWindowToClose(Unknown Source)
        at com.googlecode.lanterna@3.1.1/com.googlecode.lanterna.gui2.AbstractWindow.waitUntilClosed(Unknown Source)
        at com.sparrowwallet.sparrow@1.7.0/com.sparrowwallet.sparrow.terminal.SparrowTerminal.init(Unknown Source)
        ... 3 common frames omitted
```

**Comment by caramel-chirping-decree** [comment]
I guess I should add that Sparrow closes and this is the message in the terminal after the fact.
Exception in Application init method

**Comment by craigraw** [comment]
Thanks for the bug report, fixed in 96c88b74.

In the Preferences, what are your exchange source and currency fields set to? You can turn off fiat exchange rates by selecting an exchange rate source of None, which should prevent this from occurring.
